scripts/generate_llms.sh

```
@@ -0,0 +1,708 @@
+      cat "$1" >> "$OUTPUT_SINK"
+    fi
+  fi
+}
+
+# Iterate include dirs and collect lines
+for dir in $INCLUDE_DIRS; do
+  # If an explicit file path is provided (e.g., Package.swift, README.md), handle it directly
+  if [[ -f "$dir" ]]; then
+    file="$dir"
+    abs_file="$(cd "$(dirname "$file")" && pwd -P)/$(basename "$file")"
+    rel="${abs_file#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)